### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Dorcas126**
In this project, I implemented the CRUD (create, read, update, delete) methods. All the elements of the user interface will be fully functional and your application will be completed.

# Learning objectives
Use webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,38 +1,119 @@
...
+    this.todos = this.todos.filter((t) => t !== todo);
+    this.todos.forEach((t, i) => {
+      t.index = i;
+    });
+  }
+
+  save() {
+    localStorage.setItem('todos', JSON.stringify(this.todos));
+  }
+
```
Comment (mikethreels) [inline src/index.js]: - [ ] when trying to run the project locally as described in the readme file, the JS functionality is not working. in the console the user receives the below error:
afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Dorcas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job writing the crud methods.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,38 +1,90 @@
...
-    description: 'Drink coffee',
-    completed: true,
-    index: 3,
-  },
-];
+taskInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    if (taskInput.value === '') {
+      e.preventDefault();
+    } else {
```
Comment (V-Blaze) [inline src/index.js]: - [ ] You have done great so far, but there are still some issues you need to fix before moving to the next milestone. :+1: :100: 
- [ ] Just as the previous review pointed out on an error preventing the application from running, You can see from the video below, when I start the server, I get an error pointing to an unknown import in your CSS, the application opens when I comment out the import but the add or remove functionality doesn't run, you could see a lot of errors in the console. I'll suggest you carefully go through your project and rectify these errors. :+1: :smile: 

- - Video Reference:

https://github.com/Dorcas126/To-Do-List/assets/107632780/9c182916-75da-464d-95e9-7f8e1376ae86




- [ ] As required by Microverse, you are not meant to use any linters disable rule in your project. I'll suggest you remove the disable cmd in `line 1` of this file and try to rectify the linters issue. :+1: :100:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Dorcas126 , 

# Almost there 🟡 

1094a3e620ebea4cd6cf2d0aceaa8f52


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
-  Linters setup correctly✔
- Correct Github flow💯 🥇 ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @Dorcas126  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Add and remove functionality works great💯
-  UI looks good👍
-  Code divided into modules🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**